### ISSUE TITLE
high CPU consumption

### ISSUE THREAD

**Issue by AnonymousWebHacker** [prayag17/JellyPlayer#207]
OS: Ubuntu 22.04
CPU: Intel® Core™ i5-3360M × 4
Installed: jellyplayer.deb

**Note:** 
That is the consumption simply with the application open, without playing or transcoding anything.
You may notice the webkit2-gtk process which has multiple process threads and CPU consumption

![Captura desde 2023-10-22 12-17-49](https://github.com/prayag17/JellyPlayer/assets/43164089/1b84662e-298f-4997-afe5-1e5a5dd07fc6)

![Captura desde 2023-10-22 12-23-27](https://github.com/prayag17/JellyPlayer/assets/43164089/51b91c1a-9ddd-47a1-957b-ad852d7befd9)

**Comment by prayag17** [comment]
The builds still have the devtools enabled since they are not fully stable, which causes the cpu usage and also there are tons of unused js and bon efficient code that needs to be removed.

**Comment by AnonymousWebHacker** [comment]
Any update planned? whether stable or beta?

**Comment by AnonymousWebHacker** [comment]
I know it's beta but they are just errors that occurred to me, I hope they help you debug.

Observations:
**Windows**
I tried the Windows version, and it works relatively well, a little slow but it works, as you mentioned, it's a beta and it's fine.

**Linux Deb and Appimage**
Here my experience was different.
I was able to notice that if I open the app and simply leave it at the default sales size, the interface loads relatively smoothly.
If I widen the window, I get the initial behavior of the thread "high CPU consumption" and the interface freezes, leaving it very slow and unusable.
